classes/main.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+
+import time
+
+'''
+
+In this game we have two role - invader and hero. Both can bullet.
+
+Invaders are located at the top of game field and always move from the right side to the left and revert.
+Also they slowly move to the bottom.
+
+Main hero is located at the centre at the bottom. He can move to the left and to the right.
+If he bullet some invader, his bonus is grow.
+When invader bullet to main hero, the main hero's life is decrease.
+
+'''
+
+
+class GameScreen(object):
```

Comment:
I don't see any sense in having separate object for screen playground. Having GameController must be enough